bin/cdklocal

```diff
@@ -363,16 +363,36 @@ const applyPatches = (provider, CdkToolkit, SDK, ToolkitInfo, patchAssets = true
 };
 
 const patchPre_2_14 = () => {
-  const provider = require("aws-cdk/lib/api/aws-auth");
+  var provider = null;
+  try {
+    provider = require("aws-cdk/lib/api/aws-auth");
+  } catch(e) {
+    if (e.code == "MODULE_NOT_FOUND") {
+      console.log(e);
+      console.log("`aws-cdk` module NOT found! Have you tried to add it to your `NODE_PATH`?");
+      process.exit(1);
```

Comment:
Any reason for now just propagating the error via `throw e`? If this would 
```suggestion
      throw e;
```

nit:might be a bit cleaner in terms of error handling here, but if that led to some issues I'm not seeing, also happy to keep it as-is!